.github/workflows/release-npm-latest.yaml

```
@@ -52,4 +51,17 @@ jobs:
+  dispatch:
+    name: Dispatch to Target Repository
+    runs-on: ubuntu-latest
+    needs: publish-npm
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.TOKEN_GITHUB }}
+          repository: blinq-io/ai-qa
+          event-type: trigger-cucumber-main
+          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)